README.md

```diff
@@ -201,6 +201,7 @@ The loss functions available for training are dependent on the selected dataset
 * **Classification.** binary_cross_entropy (default), mcc (a soft version of Matthews Correlation Coefficient), dirichlet (a.k.a. evidential classification)
 * **Multiclass.** cross_entropy (default), mcc (a soft version of Matthews Correlation Coefficient)
 * **Spectra.** sid (default, spectral information divergence), wasserstein (First-order Wasserstein distance a.k.a. earthmover's distance.)
+* **Quantile Loss.** quantile_interval (Pinball loss, specify margins with `--alpha <float>`.)
```

Comment:
This listing in the readme is structured as

* **dataset_type.** list of loss functions for that dataset type

The quantile loss is not its own dataset type, it should be an entry in the regression list as "quantile_interval". Because it is one of those loss functions that enables uncertainty performance later, you should mention it in L206 after the not about `mve` and `evidential`.